### ISSUE TITLE
Improvement request: recognise new JsonMapper.builder() for Jackson ObjectMapper rules

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#212]
Instead of using `new ObjectMapper()`, also this construction can be used (since version 2.10):

```java
    private static ObjectMapper objectMapper = JsonMapper.builder()
            .enable(JsonParser.Feature.ALLOW_SINGLE_QUOTES)
            .build();
```

Check if this also triggers existing rules for ObjectMapper.

**Comment by jborgers** [comment]
@stokpop is this done?
